src/components/ec-alert/ec-alert.story.js

```diff
@@ -0,0 +1,104 @@
+import { storiesOf } from '@storybook/vue';
+import { boolean, text } from '@storybook/addon-knobs';
+import EcAlert from './ec-alert.vue';
+
+const stories = storiesOf('Alert', module);
+
+stories.add('basic', () => ({
+  components: { EcAlert },
+  props: {
+    title: {
+      default: text('title', 'Error!'),
+    },
+    subtitle: {
+      default: text('subtitle', 'Something was wrong with the update.'),
+    },
+    type: {
+      default: text('type', 'error'),
+    },
+    textButton: {
+      default: text('textButton', 'Read more'),
+    },
+    dismissable: {
+      default: boolean('dismissable', true),
+    },
+  },
+  data() {
+    return { show: true };
+  },
+  template: `
+    <div>
+    <ec-alert :title="title" :subtitle="subtitle" :type="type" :dismissable="dismissable" :button-text="textButton" v-model="show">
+    </ec-alert>
+
+    <button @click="show = !show">{{ show ? 'Hide' : 'Show' }}</button>
+    </div>`,
+}));
+
+stories.add('all', () => ({
+  components: { EcAlert },
+  data() {
+    return {
+      showInfo: true,
+      showSuccess: true,
+      showWarning: true,
+      showError: true,
+      showInfo1: true,
+      showSuccess1: true,
+      showWarning1: true,
+      showError1: true,
```

Comment:
If you have to use numbers then rename `showInfo` > `showInfo1` and `showInfo1` > `showInfo2` (bad naming conventions should be consistent too :))